rust-toolchain

```diff
@@ -0,0 +1 @@
+1.76
```

Comment:
Pinning causes other problems. We can use rust-version in Cargo.toml for MSRV if we are to bump it.